monkey/infection_monkey/system_info/windows_cred_collector/mimikatz_cred_collector.py

```diff
@@ -6,7 +6,7 @@
     WindowsCredentials,
 )
 
-LOG = logging.getLogger(__name__)
+logger = logging.getLogger(__name__)
```

Comment:
Same as above.

Reply:
Nice catch, but let's merge the PR for now, or else we'll have to re-base it a lot